content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -865,9 +895,8 @@ behavior.
 
 #### Folders
 
-- Integration directory (`cypress/e2e/` by default)
+- E2E directory (`cypress/e2e/` by default)
```

Comment:
This section is no longer accurate. I'm waiting for more information here https://cypressio.slack.com/archives/C02560911D0/p1644334073945509

Reply:
I'm going to resolve this comment, I created a follow-up story to address it. https://cypress-io.atlassian.net/browse/UDX-121